STT/whisper_stt_handler.py

```diff
@@ -24,12 +33,18 @@ def setup(
         device="cuda",
         torch_dtype="float16",
         compile_mode=None,
+        language=None,
         gen_kwargs={},
     ):
         self.device = device
         self.torch_dtype = getattr(torch, torch_dtype)
         self.compile_mode = compile_mode
         self.gen_kwargs = gen_kwargs
+        if language == 'None':
```

Comment:
I had it first set as None, but I think the performance is considerably better when we set a language. And as we support more and more languages, this can be an issue (whisper is not that good at detecting the spoken language). I think steering the users towards selecting one (or in the future a few) languages will generally improve their experience.